src/qt/bitcoinstrings.cpp

```diff
@@ -219,7 +219,7 @@ QT_TRANSLATE_NOOP("bitcoin-core", ""
 "%s)"),
 QT_TRANSLATE_NOOP("bitcoin-core", ""
 "Username and hashed password for JSON-RPC connections. The field <userpw> "
-"comes in the format: <USERNAME>:<SALT>$<HASH>. A canonical python script is "
```

Comment:
Don't modify this file, it is "Automatically generated by extract_strings_qt.py".